Src/Classes/Ease.swift

```diff
@@ -0,0 +1,179 @@
+import Foundation
+import QuartzCore
+import Flow
+
+public final class Ease<T: Easeable> {
```

Comment:
We wouldn't wanna leave these in our project, bring them out to our fork instead. `https://github.com/HedvigInsurance/ease`

Reply:
Whoops, completely forgot that this was inline 😅